lucene/core/src/java/org/apache/lucene/search/DisjunctionMaxQuery.java

```diff
@@ -262,9 +262,9 @@ public String toString(String field) {
     return buffer.toString();
   }
 
-  /** Return true iff we represent the same query as o
```

Comment:
I've seen "iff" used before as shorthand for "if and only if".  So maybe that's what the original author here was going for?  But the docs also work without it, so it's hard to say for sure.